packages/material-ui/test/typescript/moduleAugmentation/iconButtonCustionProps.spec.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
```

Comment:
I'm not sure we need a test case for each component. I was expecting one, to assert that the pattern works was enough.